service/service.go

```diff
@@ -167,43 +185,6 @@ func DeployStatusOption(statuses chan DeployStatus) Option {
 	}
 }
 
-// computeHash computes a unique sha1 value for service.
-// changes on the names of constant configuration fields of mesg.yml will not
-// have effect on computation but extending or removing configurations or changing
-// values in mesg.yml will cause computeHash to generate a different value.
-func (s *Service) computeHash() string {
-	return xstructhash.Hash(s, 1)
-}
-
-// saveContext downloads service context to a temp dir.
-func (s *Service) saveContext(r io.Reader) error {
```

Comment:
I prefer to keep this func as a wrapper so _New_ won't get too big

Reply:
Logic from save context was moved to `api/deploy.go` so only one function Untar remain.